lib/core/src/lib/datatable/components/datatable/datatable.component.scss

```diff
@@ -288,37 +288,79 @@ $data-table-cell-min-width-file-size: $data-table-cell-min-width-1 !default;
     /* stylelint-disable-next-line no-duplicate-selectors */
     .adf-datatable-cell,
     .adf-datatable-cell-header {
-        flex: 1;
+        flex: 1 1 0%;
```

Comment:
Why are we chaning styling for datatable if the ticket should only fix accessibility changes?